cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -284,7 +284,8 @@ runQueryTip (AnyConsensusModeParams cModeParams) network mOutFile = do
     CardanoMode -> do
       let localNodeConnInfo = LocalNodeConnectInfo cModeParams network sockPath
 
-      eLocalState <- liftIO $ executeLocalStateQueryExpr localNodeConnInfo Nothing $ \ntcVersion -> do
+      eLocalState <- liftIO $ executeLocalStateQueryExpr localNodeConnInfo Nothing $ do
+        ntcVersion <- ask
```

Comment:
This monadic expression checks the node to client version manually.  It will switch to automatic checking in a future PR to be less error prone, but for now, `ntcVersion` is still accessible via `ask`.